core/metacling/src/TClingCallbacks.cxx

```diff
@@ -362,6 +362,16 @@ bool TClingCallbacks::LookupObject(const DeclContext* DC, DeclarationName Name)
    if (Name.getNameKind() != DeclarationName::Identifier)
       return false;
 
+   Sema &SemaR = m_Interpreter->getSema();
+   auto *D = dyn_cast<Decl>(DC);
```

Comment:
I don't think we can have a `DeclContext` that's not a `Decl`. Maybe just use a `cast<Decl>(DC)`.

Reply:
Indeed. Some of Clang's inheritance chains are weird...